Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -450,7 +450,7 @@ private void AttachedNode_MessageReceived(Node node, IncomingMessage message)
                     VersionPayload version = node.PeerVersion;
                     if (version != null)
                     {
-                        TimeSpan timeOffset = version.Timestamp - this.dateTimeProvider.GetUtcNow();
+                        TimeSpan timeOffset = version.Timestamp - this.dateTimeProvider.GetTimeOffset();
```

Comment:
Was that a bug?

Reply:
not really, but i was quite strict here in this PR, to really use the same types everywhere
but in general, i would rather not use DateTimeOffset anywhere, but well ... it is there, so let's be consistent